scripts/build/concatenate-javascript.sh

```diff
@@ -48,3 +52,8 @@ if [ -z "$ARGV_ENTRY_POINT" ] || [ -z "$ARGV_OUTPUT" ]; then
 fi
 
 "$BROWSERIFY" "$ARGV_ENTRY_POINT" --node --outfile "$ARGV_OUTPUT"
+
+if [ "$ARGV_MINIFY" == "true" ]; then
+  uglifyjs --compress --output "$ARGV_OUTPUT.MIN" -- "$ARGV_OUTPUT"
+  mv "$ARGV_OUTPUT.MIN" "$ARGV_OUTPUT"
```

Comment:
Are we sure that `$ARGV_OUTPUT.MIN` _definitely_ contains all we need it to, before we clobber the previous `$ARGV_OUTPUT` directory?

P.S. I just did a test using some temp directrories in my /tmp folder on Linux, and `mv dir1 dir2` actually moves dir1 inside dir2? Just checking... ;-)